src/lib/storage/table.cpp

```diff
@@ -117,6 +117,11 @@ ColumnID Table::column_id_by_name(const std::string& column_name) const {
 void Table::append(const std::vector<AllTypeVariant>& values) {
   auto last_chunk = !_chunks.empty() ? get_chunk(ChunkID{chunk_count() - 1}) : nullptr;
   if (!last_chunk || last_chunk->size() >= _max_chunk_size) {
+    // One chunk reached its capacity just now. We prepare this chunk for MVCC optimizations.
+    if (last_chunk && _use_mvcc == UseMvcc::Yes) {
```

Comment:
Yes, because a chunk is either created for an empty table or for a non-empty one where the last chunk got filled up. Checking for `!_chunks.empty()` would just be a repetition of line 118. What do you suggest?